### PR TITLE
Rename `ilm set` as `ilm edit`

### PR DESCRIPTION
**Pull request by poornas**
Use label set in `mc admin bucket remote add` as storage-class for transition

### CODE REVIEW

File: cmd/ilm-add.go [hunk 1]
```diff
@@ -57,6 +57,10 @@ EXAMPLES:
   3. Add expiry and transition days rules on a prefix in mybucket.
      {{.Prompt}} {{.HelpName}} -expiry-days "300" --transition-days "200" \
           --storage-class "GLACIER" myminio/mybucket/doc
+
+  4. Add expiry and transition days rules on a prefix in mybucket for a transition tier specifief by ARN.
+     {{.Prompt}} {{.HelpName}} -expiry-days "300" --transition-days "200" \
```
Comment (vadmeste) [inline cmd/ilm-add.go:62]: s/-expiry-days/--expiry-days/

File: docs/minio-client-complete-guide.md [hunk 2]
```diff
@@ -1257,9 +1257,10 @@ USAGE:
   mc ilm COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]
 
 COMMANDS:
-  list    pretty print bucket lifecycle configuration
+  ls      lists lifecycle configuration rules set on a bucket
```
Comment (vadmeste) [inline docs/minio-client-complete-guide.md:1260]: s/lists/list/

File: go.mod [hunk 3]
```diff
@@ -2,6 +2,8 @@ module github.com/minio/mc
 
 go 1.13
 
+replace github.com/minio/minio => ../minio
```
Comment (vadmeste) [inline go.mod:5]: To remove

**Review by vadmeste — COMMENTED** [review]
Some comments

**Review by BigUstad — APPROVED** [review]
LGTM. Tested

File: go.mod (hunk 3 above)
Comment (kannappanr) [reply go.mod:5]: is this still required?

Comment (poornas) [reply go.mod]: removed.

**Review by kannappanr — APPROVED** [review]
LGTM

**Comment by kannappanr** [comment]
@poornas Please resolve the conflicts

**Comment by poornas** [comment]
needs https://github.com/minio/minio/pull/10565 to merge first for pkg/madmin changes.

File: cmd/ilm-add.go [hunk 4]
```diff
@@ -83,7 +88,7 @@ var ilmAddFlags = []cli.Flag{
 	},
 	cli.StringFlag{
 		Name:  "storage-class",
-		Usage: "storage class for transition (STANDARD_IA, ONEZONE_IA, GLACIER. Etc)",
+		Usage: "storage class for transition (STANDARD_IA, ONEZONE_IA, GLACIER. Etc).For setting up transition to a MinIO target, use the label defined for the target",
```
Comment (vadmeste) [inline cmd/ilm-add.go:91]: Missing space after '.' here

**Review by vadmeste — COMMENTED** [review]
One typo, LGTM otherwise

**Pull request merged**